docs/en/backup-restore.rst

```diff
@@ -2,56 +2,169 @@
 
 .. _backup-restore.rst:
 
-backup-restore.rst
-+++++++++++++++++++++++++++
+Backup and Restore
+++++++++++++++++++
+The **Backup and Restore** functionality becomes relevant when the User can no longer access the Digital Credentials stored on the mobile device where the Wallet Instance was installed. Situations that may require the User to utilize this functionality include:
+
+- The mobile device has been **lost**, **stolen**, **damaged**, or **compromised** (e.g., due to unauthorized access).
+- The User replaces an existing Wallet Instance with a new instance of the same Wallet Solution.
+- The User has changed their mobile device and needs to set up the Wallet Solution on the new device.
+- The User performs a factory reset on the current phone and needs to set up the Wallet Solution again.
+
+.. note::
+  
+  The Backup and Restore functionality is different from migrating to another Wallet Solution (a.k.a. Data portability). 
+  In the latter case, we are dealing with a scenario in which the User wants to migrate from his current Wallet Solution to a different one due to ceases to support the Wallet Solution as highlighted in Annex 2 of `ARF`_.    
+
+Backup Flow
+-----------
+
+.. _fig_Backup_flow:
+.. figure:: ../../images/Backup_flow.svg
+   :name: Sequence Diagram for Wallet Instance Backup
+   :alt: The figure illustrates the sequence diagram for backup flow, with the steps explained below.
+   :target: https://www.plantuml.com/plantuml/png/VP5HRzGm3CVVyodClMn8j1KmU9YcqxPZe8bDEkaqyG1eIbFlQaYJAd4uAhJlJj96WuvZUMhj_y_-spxrB1s7JejdP9GE3KBBtFlZgd9oLsw9sr07ZqvPmsYuLBQhUYrDOWhFZQQwMXqLwnIwkRwgEkaPNGpThY8XoQ0h-rHVICNMmKsi1TB38dqiXDWCKT_TdjjW6kc6mvtK6dbZTM2oviMaE_3m3d-GmiLp-2KWlltOfV4iZSA8VHe3a2CPpE_1sM6Wt24A6TsTJCezkbggxw4_wsc3Blc8rFaOWhFr9UHW9k_5dEriJHetR9tS9l1w_Cy3mPLLKaFE9fUvnhqG1t3nizUaY47BmGO6sNmBdZiq37VMGMyzfIsHsOfP5oW-jzGqQBuMIvYlHeXnt28c0i4nB4xgvSiIyZGhXv5YajgVLFLo8QBc96aVBs02NvNm0GqwoGWVSO1rwwJ7FsWYKxj9_ReSvzmZVLmT_j_og4mcKyCBezpGCpQGpRydZK_K2pHLU5F-Y-vp_8GKlXY8QTWHjx1sjkkdW_oL6-zQhRGDJRvkzlQm_ld5fePlInZ1ENAAfWcT_Wq0
+   
+   Backup flow.
+
+Below, the description of the steps of :numref:`fig_Backup_flow`:
+
+**Step 1**: The User clicks on the backup credentials option in the Wallet Instance. 
+
+**Steps 2-3**: The Wallet Instance using the Backup APIs randomly selects 10 key phrases from a pre-generated list of words and displays it to the User. 
+The User MUST securely store the key phrases (e.g., in a password manager or a physical safe) as they are critical for restoring the backup.
```

Comment:
```suggestion
The User MUST securely store the key phrases (e.g., in a password manager or a physical safe) as they are critical for restoring the backup.
```
I am not sure that this is the best secure way to keep the password safe.

Reply:
It is up to the user to decide where to save their passwords, it cannot be regulated